### ISSUE TITLE
MCF language: negation in if-then-else

### ISSUE THREAD

**Issue by RolandKluge** [eMoflon/emoflon-tie-gt#18]
Maybe an 'unless' construct could avoid the following hack.
```
    this: Graph
    srcNode : Node
    trgNode : Node
    if (pNodeExists(nodeId = srcId, graph = this)) {} else {
      invoke pCreateNode(nodeId = srcId, graph = this, newNode = srcNode)
    }
```

Desired:
```
    this: Graph
    srcNode : Node
    trgNode : Node
    unless (pNodeExists(nodeId = srcId, graph = this))  {
      invoke pCreateNode(nodeId = srcId, graph = this, newNode = srcNode)
    }
```

**Comment by RolandKluge** [comment]
Won't implement

**Comment by RolandKluge** [comment]
Solved in 72bd0c1665d929f297e5ceb8d995a2dbcd2d8c0e
